### PR TITLE
add tests for stopping projections and queries, fix small bug

### PR DESCRIPTION
**Pull request by prolic**
see: https://github.com/prooph/pdo-event-store/issues/10

### CODE REVIEW

File: tests/Projection/MySQLEventStoreReadModelProjectionTest.php [hunk 1]
```diff
@@ -209,4 +209,42 @@ public function it_throws_exception_on_run_when_nothing_configured(): void
...
+
+                if ($state['count'] === 10) {
+                    $this->stop();
+                }
+
+                return $state;
+            })
+            ->run();
+
+        $this->assertEquals(10, $projection->getState()['count']);
```
Comment (basz) [inline tests/Projection/MySQLEventStoreReadModelProjectionTest.php]: no test on the model?

File: tests/Projection/PostgresEventStoreReadModelProjectionTest.php [hunk 2]
```diff
@@ -209,4 +209,42 @@ public function it_throws_exception_on_run_when_nothing_configured(): void
...
+
+                if ($state['count'] === 10) {
+                    $this->stop();
+                }
+
+                return $state;
+            })
+            ->run();
+
+        $this->assertEquals(10, $projection->getState()['count']);
```
Comment (basz) [inline tests/Projection/PostgresEventStoreReadModelProjectionTest.php]: idem?

**Comment by prolic** [comment]
@basz I updated the read model projection tests

**Pull request merged**